src/js/components/FilterByService.js

```diff
@@ -93,8 +93,9 @@ var FilterByService = React.createClass({
         dropdownMenuListItemClassName="clickable"
         wrapperClassName="dropdown"
         items={this.getDropdownItems()}
-        onItemSelection={this.handleItemSelection}
         initialID={this.getSelectedId(this.props.byServiceFilter)}
+        onItemSelection={this.handleItemSelection}
+        ref={(ref) => this.dropdown = ref}
```

Comment:
why don't you just make this `ref` a string and you can just access it by doing `this.refs.dropdown`